app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1> Quotes Search Results </h1>
+<h2> Total Results: <%= @quotes.count %> </h2>
+<% @quotes.each do |quote| %>
+    <h3> Quote: <%= quote.content %> </h3>
+    <h3> Author: <%= quote.author %> </h3>
+    <h3> Tags: <%= quote.tags %> </h3>
```

Comment:
It would be a little more user friendly in the view if you either joined the tags together as a string first or enumerated over the list so they don't just show up as an array on the page.